automation/project/Dependencies.scala

```diff
@@ -26,6 +33,12 @@ object Dependencies {
     "org.specs2"          %%  "specs2-core"   % "3.7"  % "test",
     "org.scalatest"       %%  "scalatest"     % "2.2.6"   % "test",
     "org.seleniumhq.selenium" % "selenium-java" % "2.35.0" % "test",
-    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0"
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
+
+    workbenchModel,
+    workbenchGoogle,
+
+    // required by workbenchGoogle
```

Comment:
Shouldn't workbench-google pull this in as a transitive dependency? Or are you having to intervene to get the right version?

Reply:
It's marked as `provided` in workbench-google, so downstream systems must pull it in explicitly.

Actually I'm confused why it's `provided` here too -- who is actually providing the dependency?

(or is it not provided at all, and this is just to make the build work?)

Reply:
I haven't thoroughly investigated this.  Without this line, I get a runtime error: `spray/json/JsonWriter` class not found

Reply:
ok, fine to leave as is then